lms/djangoapps/discussion/views.py

```diff
@@ -63,16 +63,27 @@ def make_course_settings(course, user):
 
 
 @newrelic.agent.function_trace()
-def get_threads(request, course, discussion_id=None, per_page=THREADS_PER_PAGE):
+def get_threads(request, course, user_info, discussion_id=None, per_page=THREADS_PER_PAGE):
     """
     This may raise an appropriate subclass of cc.utils.CommentClientError
     if something goes wrong, or ValueError if the group_id is invalid.
+
+    Arguments:
+        request: The user request.
+        course: The course object.
+        user_info: The comment client User object as a dict.
+        discussion_id: Optional discussion id/commentable id for context.
+        per_page: Optional number of threads per page.
+
+    Returns:
+        A tuple of the threads and the query parameters used for the search.
+    :return:
```

Comment:
Thanks. I just missed this part of the cleanup of PyCharm's auto-generated docstring. You ok with the missing type designations? I was looking to make things a little nicer, without necessarily digging that deep on existing parameters.